csrc/fusion_segmenter.cpp

```diff
@@ -1822,12 +1822,12 @@ std::string toString(const SegmentedFusion* segmented_fusion) {
   return ss.str();
 }
 
-//! Sets the logical as root and erases logical of all inputs in fusion. Any
+//! Sets the root as logical and erases root of all inputs in fusion. Any
 //! non-constant expressions in those extents are replaced by new scalars with
 //! no definition. These mutations are performed throughout the Fusion so that
 //! downstream expressions dependent on the original inputs' logical extents can
 //! be computed properly.
-void convertInputLogicalsToRoots(Fusion* fusion) {
+void eraseInputDistinctRootDomains(Fusion* fusion) {
```

Comment:
I'm open to suggestions on the new name of this function but since we are actually clearing out the root domain now, I think it should be changed.